NewForce-Cohort-9/automated-hello-world-1989s
# Il-portugal-2: CSS stylings & HTML update for portugal page

Added CSS changes (portugal.css)

- added background image
- updated font colors
- used flexbox for layout
- added borders to each section 

Portugal HTML (portugal.html)
- links main CSS and portugal CSS files
- added NAV bar 

To test

1.  Switch to my branch (il-portugal-2)
2. run serve in terminal
3.  see if page loads with changes
4.  leave a comment if it looks good! If incorrect, send me a message!
